superdesk/publish/publish_queue.py

```diff
@@ -116,6 +132,45 @@ def _set_queue_state(self, updates, original):
             else updates.get("state") or QueueState.PENDING.value
         )
 
+    def is_updated_document(self, doc):
+        if not doc.get("item_id"):
+            return
+        article = get_resource_service("published").find_one(req=None, guid=doc["item_id"])
```

Comment:
not sure this will give you latest item in general, it can be any version of that published item